port_ocean/clients/port/mixins/entities.py

```diff
@@ -58,7 +58,7 @@ async def upsert_entity(
             )
         handle_status_code(response, should_raise)
         result = response.json()
-        result_entity = Entity.parse_obj(result)
+        result_entity = Entity.parse_obj(result.get("entity", {}))
```

Comment:
is it a bug fix? as until now we didn't passed the `"entity"` key?

Reply:
Correct